django/db/backends/base/features.py

```diff
@@ -376,6 +376,9 @@ class BaseDatabaseFeatures:
     # Does the backend support native tuple lookups (=, >, <, IN)?
     supports_tuple_lookups = True
 
+    # Does the backend support native tuple gt(e), lt(e) comparisons in subqueries?
+    supports_tuple_comparison_in_subquery = True
```

Comment:
Not a native speake but maybe _against_ is a better name?

```suggestion
    supports_tuple_comparison_against_subquery = True
```